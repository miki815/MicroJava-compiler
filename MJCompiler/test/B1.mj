
program test301
	int niz[]; int x, y, z;
{
	void main()	
	{
		x = 55;
		niz = new int[3];
		niz[0] = 815;
		niz[1] = 816;
		niz[2] = 817;
		niz.foreach(x => {
			print(x);
		});
		print(333);
		x = 5;
		y = 10;
		while(x < y){print(x); x++; z = 2; while(z > 0) {print(z); z--;}}
	}
}


