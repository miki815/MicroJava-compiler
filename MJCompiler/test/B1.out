  815  816  817  333    5    2    1    6    2    1    7    2    1    8    2    1    9    2    1
Completion took 2 ms